### ISSUE TITLE
Updating `document.title` on history change

### ISSUE THREAD

**Issue by tilsammans** [codevise/pageflow#833]
What do you think about changing the `document.title` (either via JS or actually in the DOM) on page change? It would allow client-side trackers to track the new title.

Would all three history adapters need to be changed?

**Comment by tf** [comment]
Look at [the MDN docs for `pushState`](https://developer.mozilla.org/en-US/docs/Web/API/History_API#The_pushState()_method) the title parameter appears to be ignored. So maybe setting `window.title` somewhere might be enough? In #831 I made a change, though, to prevent history from being recorded when scrolling through pages. We got a lot of feedback (and also felt ourselves) that not being able to easily leave an entry via the back button was annoying. Not quite sure how this affects the change suggested here.

For client side tracking we mostly report things explicitly (listening to `pageflow.events.on('page:change', () => {})`).
